mungegithub/github/github.go

```diff
@@ -147,7 +147,7 @@ type Config struct {
 	State  string
 	Labels []string
 
-	token     string
+	Token     string
```

Comment:
this was purposely private so it won't get printed out in the logs-- can you add a comment saying this and change it back?